package.json

```diff
@@ -33,7 +33,7 @@
     "core-js": "^3.21.1",
     "fast-glob": "^3.2.7",
     "foundation-emails": "^2.3.1",
-    "identity-style-guide": "^6.5.0",
+    "identity-style-guide": "^6.6.0",
```

Comment:
do we want to up to [6.7.0](https://github.com/18F/identity-style-guide/pull/330) too? or wait on that

Reply:
Yeah if I can get that published first, would be good to bring it in here.

Reply:
> do we want to up to [6.7.0](https://github.com/18F/identity-style-guide/pull/330) too? or wait on that

Updated in rebased 67e86af58.